src/devices/video/gb_lcd.cpp

```diff
@@ -3508,6 +3496,46 @@ WRITE8_MEMBER(cgb_ppu_device::video_w)
 
 // Super Game Boy
 
+/* Super Game Boys transfer VRAM data using the display signals.
+ * That means not DMG VRAM is copied from 0x8800 to SNES VRAM
+ * but displayed BG contents.
+ * Things to consider: LCDC (0xFF40) and SCY/SCX (0xFF42/3)
+ *  - LCD must be on
+ *  - TODO: Window might or might not influence result
+ *  - CHR and BG likely influence result
+ *  - BG must be on
+ *  - BG should not be scrolled, but likely does influence transfer
+ *  - TODO: are BG attributes (hflip, vflip) ignored?
+ */
+/**
+ * Copy DMG VRAM data to SGM VRAM
+ * @param dst       Destination Pointer
+ * @param start     Logical Start Tile index inside display area.
+ * @param num_tiles Number of DMG tiles (0x10u bytes) to copy.
+ */
+void sgb_ppu_device::sgb_vram_memcpy(uint8_t* dst, uint8_t start, size_t num_tiles) {
+	
+	size_t i;
+	uint16_t bg_ix = (start / 0x14u) * 0x20u + (start % 0x14u);
```

Comment:
Please use uppercase size/signedness qualifiers - it avoid potential confusion between `101` and `10l` (`10L` is more visually distinct).